debian/control

```diff
@@ -810,12 +826,16 @@ Breaks: mariadb-oqgraph-engine-10.0,
         mariadb-oqgraph-engine-10.1,
         mariadb-oqgraph-engine-10.2,
         mariadb-oqgraph-engine-10.3,
-        mariadb-oqgraph-engine-10.4
+        mariadb-oqgraph-engine-10.4,
+        mariadb-server-10.0,
+        mariadb-server-10.1
 Replaces: mariadb-oqgraph-engine-10.0,
           mariadb-oqgraph-engine-10.1,
           mariadb-oqgraph-engine-10.2,
           mariadb-oqgraph-engine-10.3,
-          mariadb-oqgraph-engine-10.4
+          mariadb-oqgraph-engine-10.4,
+        mariadb-server-10.0,
```

Comment:
Is here wrong spaces/tabs?

Reply:
Thanks for noticing! Fixed now with `wrap-and-sort -av`